data/json/items/comestibles/med.json

```diff
@@ -402,5 +402,46 @@
         "charges" : 20,
         "stack_size" : 60,
         "fun" : 5
+    },
+    {
+        "id" : "weed",
+        "type" : "COMESTIBLE",
+        "name" : "marijuana",
+        "name_plural" : "marijuana",
+        "weight" : 1,
+        "color" : "green",
+        "use_action" : {
+            "type" : "consume_drug",
+            "activation_message" : "You smoke some weed.  Good stuff, man!",
+            "effects" : [ {
+                    "id" : "weed_high",
+                    "duration" : 90
+                },
+                {
+                    "id": "pkill1",
+                    "duration": 60
+                }
+            ],
+            "stat_adjustments": {
+                "hunger" : 4,
+                "thirst" : 6
+            },
+            "fields_produced" : {"fd_weedsmoke" : 2},
+            "charges_needed" : { "fire" : 1 },
+            "tools_needed" : { "apparatus" : -1 },
+            "moves" : 40
+        },
+        "stim" : -10,
+        "comestible_type" : "MED",
+        "symbol" : "!",
+        "quench" : -5,
+        "healthy" : 0,
```

Comment:
I'm going to leave this here for now. I'd like to change it but I want to change it with the joint at the same time.